Fix warning Umbrella header for module

wowpotato opened the pull request:
When installing SVProgressHUD via sub-framework (also SwiftPM), I received the following warning:

```bash
Umbrella header for module 'SVProgressHUD' does not include header 'SVIndefiniteAnimatedView.h'
Umbrella header for module 'SVProgressHUD' does not include header 'SVProgressAnimatedView.h'
Umbrella header for module 'SVProgressHUD' does not include header 'SVRadialGradientLayer.h'
```
Umbrella header such as 'SVProgressHUD.h' should import these additional headers to resolve the warning.

This fixes that warning.

@honkmaster 
@tobihagemann 
@juliensaad

Commit:
Fix warning Umbrella header for module

Update CHANGELOG.md

Fix warning Umbrella header for module

wowpotato commented:
@juliensaad Could you please review and merge this pull request as soon as possible? Thank you!

vectorstofinal commented:
I'm seeing this issue too - but as an error - this PR fixed it for me!

hujunfeng commented:
I believe adding these headers in SVProgressHUD.h is unnecessary.

The 3 classes are not public, therefore they should not be added to the "include" folder in the first place. The proper fix to the warnings is to remove the 3 symbolic links of the headers from the "include" folder.

wowpotato commented:
> I believe adding these headers in SVProgressHUD.h is unnecessary.
> 
> The 3 classes are not public, therefore they should not be added to the "include" folder in the first place. The proper fix to the warnings is to remove the 3 symbolic links of the headers from the "include" folder.

Thank you for pointing that out. 
After reviewing your feedback, I agree that including these headers in SVProgressHUD.h is unnecessary. 
The classes are not public and should not be in the "include" folder. 
I have updated the PR to reflect this change and removed the symbolic links as suggested.

hujunfeng commented:
@wowpotato Thanks for making the changes. I hope the PR gets accepted soon.

alex-vasenin commented:
Will it ever be merged?

godLoveY commented:
您好，邮件已收到

TunvirRahman commented:
requesting merge.

godLoveY commented:
您好，邮件已收到

michael-konexus commented:
merge please 👍 👍 👍
